services/tests/test_services_graphql_api.py

```diff
@@ -51,3 +53,33 @@ def test_normal_user_can_add_service_mutation(rf, user_gql_client):
     }
     executed = user_gql_client.execute(query, context_value=request)
     assert dict(executed["data"]) == expected_data
+
+
+def test_normal_user_cannot_add_service_multiple_times_mutation(rf, user_gql_client):
+    request = rf.post("/graphql")
+    request.user = user_gql_client.user
+    ProfileFactory(user=user_gql_client.user)
+
+    t = Template(
+        """
+        mutation{
+            addService(service: {type: ${serviceType}}){
+                service{
+                    type
+                }
+            }
+        }
+        """
+    )
+    creation_data = {"serviceType": "BERTH"}
+    query = t.substitute(**creation_data)
+    expected_data = {
+        "addService": OrderedDict({"service": OrderedDict({"type": "BERTH"})})
```

Comment:
does it have to be `OrderedDict` here? you cast it to dict anyway a couple of lines later: `dict(executed["data"])`